smart-contracts/contracts/cl-vault/src/vault/any_deposit.rs

```diff
@@ -84,13 +83,21 @@ pub fn execute_any_deposit(
         // rest minting logic remains same
         Ok(Response::new()
             .add_submessage(SubMsg::reply_on_success(
-                swap_msg,
+                swap_calc_result.swap_msg,
```

Comment:
We create the same message below in line 129. We could just read the data for the attributes in the if-else-branch and then have one return statement instead of one for each branch